logging/fb/viya-tracing.conf

```diff
@@ -0,0 +1,12 @@
+# Version 1.9.9
+[INPUT]
+   name             opentelemetry
+   listen           0.0.0.0
+   port             4318
+   tag              viya_otel
+[OUTPUT]
+   Name opentelemetry
+   Host  v4m-tempo.monitoring
```

Comment:
Hard-coded reference to "monitoring" namespace.  Can we just omit the namespace? Or, do we need to do some sort of substitution?

Reply:
I think I can remove, let me test this.